source/includes/fundamentals/code-examples/linq.cs

```diff
@@ -63,3 +63,56 @@ public class Review
 }
 
 // end-review-model
+
+// start-ingredient-model
+
+public class Ingredient 
+{
+    public int Id { get; set; }
+
+    public string Name { get; set; }
+
+    public int Price { get; set; }
+
+    public int Count { get; set; }
+}
+
+// end-ingredient-model
+
+// start-bitAnd-example
+
+var query = queryableCollection
+    .Where(i => i.Name == "eggs")
+    .Select(i => i.Price & i.Count);
+
+// end-bitAnd-example
+
+// start-bitAnd-collection-example
+
+var query = queryableCollection
+    .Select(i => i.Price & i.Count);
```

Comment:
This is an odd example. There's no real reason to bit-and a `Price` and a `Count`.

A better example would involve flags or something.

Reply:
Changed the example!